README.md

```diff
@@ -1,2 +1,63 @@
 # Leaderboard
-Leaderboard list app, using webpack and ES6 modules.
+Leaderboard list app, using webpack and ES6 modules.  The projeect consume the Leaderboard API using JavaScript async and await methods.
+
+![Leaderboard](../leader.png)
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- Git
+- Webpack
+
+## Getting Started
+
+In order to run this project on your computer, please make sure that you have a web browser installed. You can clone the repo to your machine.
+
+### Prerequisites
+Node package Manager(NPM)
+To install NPM on your local machine, please visit NPM official website
+Text editor(Vscode, Atom, NotePad++, Sublime)
+
+- Click on the clone button to copy the clone
+- Open Terminal and run git clone <copied address>
+- Clone the repo
+- From you terminal run " code ." to open in your code editor.
+
+### Install
+Run the following commands to install dependencies
+
+`
+npm install
+`
+### Usage
+`
+npx hint .
+npx stylelint "**/*.{css,scss}"
+npx eslint .
+`
+
+## Authors
+👤 **Promise Uka**
+
+- GitHub: [@Ukaypromise](https://github.com/Ukaypromise/)
+- Twitter: [@PromiseUkay](https://twitter.com/PromiseUkay)
+- LinkedIn: [@promiseuka](https://www.linkedin.com/in/promiseuka)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Please, kidly add the missing license content as the link is broken to make your readme more professional.